docs/developer/05-notification-plugins.md

```diff
@@ -31,6 +31,14 @@ and passed in two sets of Map data:
 
 The Configuration data is fully custom depending on your plugin, and is described in the [Plugin configuration properties](#plugin-configuration-properties) section.
 
+## Notifications thread
+
+Notifications can be run on their own thread by setting the following properties in the rundeck-config.properties file:
+
+`rundeck.config.notifications.own.thread`: ('true','false') When set to 'true' it will trigger the notifications on a separate thread.
+`rundeck.config.notifications.timeout`: (numeric, in miliseconds) When you set the notifications to be triggered on a separate thread, you can also set a time out for the notification thread so that it must be sent before a certain time.
```

Comment:
Is there a suggested value?  Is there a default value if this isn't specified?

Reply:
There is no default time out, i'll add that to the doc.

Reply:
So it would never timeout?  It would go indefinitely?

Reply:
Yes it would, similary to the current behavior. The only difference is that it would not stuck the job execution. You think it would be better to have a default time out? If that's so, what you think it would be an appropiate value for it?